### PR TITLE
use nginx for certificates in admin dashboard

### PR DESCRIPTION
**Pull request by OmarElawady**
### Description

This is to make the admin dashboard peertube solution compatible with the new image changes. It now uses nginx certbot to create a test certificates and serve the given domain name.

### CODE REVIEW

File: jumpscale/packages/tfgrid_solutions/chats/peertube.py [hunk 1]
```diff
@@ -155,27 +155,7 @@ def reservation(self):
...
-            solution_uuid=self.solution_id,
-            **self.solution_metadata,
-        )
-        success = deployer.wait_workload(_id, self)
-        if not success:
-            # solutions.cancel_solution(self.workload_ids)
-            raise StopChatFlow(f"Failed to create trc container on node {self.selected_node.node_id}" f" {_id}")
-
-        entrypoint = f'/usr/local/bin/startup.sh "{self.domain}" "{self.email}"'
+        entrypoint = f'/usr/local/bin/startup.sh "{self.domain}"'
```
Comment (AhmedSa-mir) [inline jumpscale/packages/tfgrid_solutions/chats/peertube.py:158]: Why did you remove the email parameter from the command?

**Review by AhmedSa-mir — APPROVED** [review]

File: jumpscale/packages/tfgrid_solutions/chats/peertube.py (hunk 1 above)
Comment (OmarElawady) [reply jumpscale/packages/tfgrid_solutions/chats/peertube.py:158]: The only use for the email was to generate the certificate now nginx container creates it.

**Review by xmonader — APPROVED** [review]
